scripts/generate-a3p-submission.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ueo pipefail
+
+sdkroot=$(cd -- "$(dirname "$0")/.." >/dev/null && pwd)
```

Comment:
doesn't this have the same issue as motivated you to revert the previous? people can also forget to update this script. moreover, if they do manually update the submission, this will overwrite it.

however, this is (as was the previous commit imo) better than the status quo so I approve.

Reply:
> doesn't this have the same issue as motivated you to revert the previous? people can also forget to update this script.

The problem in the previous attempt was that the submission would get out of sync with the code in the repo. Here the submission is re-generated every time `a3p-integration` builds.

Yes adding or changing the shape of new a3p proposal requires to update this script, but that's much less frequent.

> moreover, if they do manually update the submission, this will overwrite it.

The submission is git excluded. If the submission needs editing for some reason, it should be automated in the script that generates it.